scripts/run_simulator.py

```
@@ -0,0 +1,19 @@
+Import("env")  # noqa: F821 - SCons injects this at build time
+
+
+def run_simulator(source, target, env):
+    import os
+    import subprocess
+
+    binary = env.subst("$BUILD_DIR/program")
+    subprocess.run([binary], cwd=os.getcwd(), check=True)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)